src/samples/flow/netctl/network-connect.fbp

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sol-fbp-runner
+
+# This file is part of the Soletta™ Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This program will print to console a list of all network services
+#
+# Examples:
+#  * connect or disconnect network services
+#    $ ./network-connect.fbp
+
+conn(netctl/service:name="Guest")
+stat(netctl/state:name="Guest")
+tick(constant/empty)
+
+tick OUT -> TRIGGER stat OUT -> IN _(console:prefix="state: ")
+tick OUT -> CONNECT conn
+conn OUT -> IN _(console)
```

Comment:
also, to make this sample more useful, you can connect to `OUT` port (to be introduced!) and when service is connected you can quit the application.